docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -75,6 +75,50 @@ ALTER TABLE table_identifier ADD COLUMNS ( col_spec [ , ... ] )
 
     Specifies the columns to be added.
 
+### DROP COLUMNS
+
+`ALTER TABLE DROP COLUMNS` statement drops mentioned columns from an existing table.
+Note that this statement is only supported with v2 tables.
+
+
```

Comment:
nit: I think one blank line. There are a couple of more places that you have an extra blank line.